### ISSUE TITLE
2 channel wall switch TZ3000_e98krvvk

### ISSUE THREAD

**Issue by Peter-Celica** [JohanBendz/com.tuya.zigbee#734]
Hi Johan after your recent update my
2 Channel wall switch
_TZ3000_e98krvvk Model TS0012
Is not responding anymore, I'll hope you can fix it again.
Thanks for all the good work again.
Best regards Peter.

**Comment by JohanBendz** [comment]
Try interviewing the device: https://tools.developer.homey.app/tools/zigbee (Action / Interview).
If the device does not interview, press the pairing button every other second during interview.

**Comment by Peter-Celica** [comment]
Hi Johan thanks for your reply, I tried to interview the wall switch but after interviewing it hangs.
I'll try it tomorrow with disconnecting the lead wire and after reconnecting to add it again.
Best regards and nice evening Peter.

**Comment by Peter-Celica** [comment]
Hi Johan good evening, I hope everything is going well with you 👍🏻 and the switch is working again after I tried to add it again, I did it while the former one was still in my Homey. While I was adding it again it said that it was already added and after that the former installed switch was working again.
It's strange but true, maybe it makes sense to you.
Best regards and have a nice weekend Peter.😋
